src/services/conversionRates/getConversionRatesService.js

```diff
@@ -50,45 +50,18 @@ const _getRatesCoinGecko = async (
   const promises = ratesToGet.map(async r => {
     const rateSymbolInner = stableCoins.includes(r) ? 'USD' : r;
     if (rateSymbolInner !== rateSymbol) {
-      const today = new Date(new Date());
-      const todayUTC = today.setUTCHours(0, 0, 0, 0);
-      const todayRounded = Math.round(todayUTC / 1000);
-      const timestampRounded = Math.round(timestampMS / 1000);
-      const differenceMillis = todayRounded - timestampRounded;
-      let differenceDays = 0;
+      const timestampTo = Math.round(timestampMS / 1000);
```

Comment:
set timestampTo later than timestampMS as timestampFrom is sooner than it.

Reply:
Checked it, timestampTo is in the future from timestampFrom now